tests/agents/arbitrage_agent/test_arbitrage_agent.py

```diff
@@ -0,0 +1,66 @@
+import typing as t
+from unittest.mock import Mock, patch
+
+import pytest
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from prediction_market_agent_tooling.markets.omen.omen import OmenAgentMarket
+
+from prediction_market_agent.agents.arbitrage_agent.deploy import (
+    DeployableOmenArbitrageAgent,
+)
+
+
+@pytest.fixture(scope="module")
+def arbitrage_agent() -> t.Generator[DeployableOmenArbitrageAgent, None, None]:
+    with patch(
+        "prediction_market_agent.agents.arbitrage_agent.deploy.DeployableOmenArbitrageAgent.load",
+        new=lambda x: None,
+    ), patch(
+        "prediction_market_agent_tooling.tools.langfuse_.get_langfuse_langchain_config"
+    ):
+        agent = DeployableOmenArbitrageAgent()
+        # needed since load was mocked
+        agent.chain = agent._build_chain()
+        yield agent
+
+
+@pytest.fixture(scope="module")
+def main_market() -> t.Generator[AgentMarket, None, None]:
+    m1 = Mock(OmenAgentMarket, wraps=OmenAgentMarket)
+    m1.question = "Will Kamala Harris win the US presidential election?"
+    yield m1
+
+
+@pytest.fixture(scope="module")
+def related_market() -> t.Generator[AgentMarket, None, None]:
+    m1 = Mock(OmenAgentMarket, wraps=OmenAgentMarket)
+    m1.question = "Will Kamala Harris become the US president in 2025?"
```

Comment:
I'm a bit confused by this, if you expect high correlation with the other question shouldn't this be either:
- `"Will Kamala Harris be the US president in 2025?"` or
- `"Will Kamala Harris become the US president in 2024?"`

Reply:
Oh scrap that sorry, elected president takes office in Jan 2025. Forgot!